appliances/hbcd_pe.gns3a

```diff
@@ -0,0 +1,64 @@
+{
+    "appliance_id": "ac98ab6f-7966-444b-842f-9507c965b8b7",
+    "name": "HBCD-PE",
+    "category": "guest",
+    "description": "Hiren’s BootCD PE (Preinstallation Environment) is a restored edition of Hiren’s BootCD based on Windows 11 PE x64. ",
+    "vendor_name": "hirensbootcd.org",
+    "vendor_url": "https://www.hirensbootcd.org/",
+    "documentation_url": "https://www.hirensbootcd.org/howtos/",
+    "product_name": "Hiren’s BootCD PE",
+    "product_url": "https://www.hirensbootcd.org/",
+    "registry_version": 4,
+    "status": "stable",
+    "maintainer": "GNS3 Team",
+    "maintainer_email": "developers@gns3.net",
+    "usage": "",
+    "symbol": "cd.svg",
```

Comment:
Is this a builtin-in symbol or do you want to provide one?

Reply:
Sorry my mistake.
I only have 
https://commons.wikimedia.org/wiki/File:CD_icon.svg
https://fr.m.wikipedia.org/wiki/Fichier:CD_icon_test.svg
https://en.m.wikipedia.org/wiki/File:CD_icon.svg
Not sure what is the most appropriate licence to start with
I think ideally we should have a computer showing a CD (either on the screen or next to keyboard).
I'm sorry but I don't have artistic drawing skills ;-)